Entities/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Entities
+{
+    public class User
+    {
+        public string UserEmailId { get; set; }
+        public int UserAge { get; set; }
```

Comment:
Just curious, what will you do with a user's age in our system?

Reply:
Ummmmm
I dont know
I just added it 
I can remove it